types/artifacts.d.ts

```diff
@@ -461,6 +463,14 @@ declare global {
         clientRects: Rect[];
       }
 
+      export interface TraceNode {
+        metricTag: string;
```

Comment:
can this just be `metric`?

Reply:
I wanted to avoid just calling it metric as it doesn't actually contain the metric it's related to. But perhaps metricName makes more sense.